### PR TITLE
RDAPI 30339 

### PR DESCRIPTION
**Pull request by jolearyaxway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

adding a known issue for cassandra startup and a workaround

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 5b32b9a |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2694.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md [hunk 1]
```diff
@@ -220,6 +220,55 @@ Following the upgrade to Jython 2.7.3, the following issue has been observed for
 
 Related Issue: RDAPI-29910
 
+### Cassandra uses a deprecated garbage collector
+
+Following the upgrade to Java 11, the following issue has been observed on startup of Cassandra - `VM warning: Option UseConcMarkSweepGC was deprecated in version 9.0 and will likely be removed in a future release`. This does not affect performance, Cassandra remains functional, however, by changing the garbage collector (GC) to `G1GC` the problem will be resolved, in addition in certain circumstances resource utilization may be improved. To change the GC one must modify the `jvm11-server.options` file located in the `conf` directory inside the root directory of the Cassandra install.
```
Comment (JohnD-Axway) [inline content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:225]: Maybe say: "the warning will be resolved" instead of problem.

**Review by JohnD-Axway — APPROVED** [review]

File: content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md (hunk 1 above)
Comment (Bcolemanaxway) [inline content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:225]: The "Cassandra remains functional" bit could probably be removed for readability IMO, the sentence is quite long. You could also separate the "in addition.." onwards into a new sentence. Up to you though, @andreamussap will probably have a better say on whether to change or not.

Comment (jolearyaxway) [reply content/en/docs/apim_relnotes/20230830_apimgr_relnotes.md:225]: reread it, I agree, it's a bit wordy, besides the reader will know warnings aren't fatal and will likely know Cassandra will still be operating fine, I'll remove this.

**Review by Bcolemanaxway — APPROVED** [review]
